.sbtopts

```diff
@@ -12,5 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
--mem 4096
+#-mem 4096
+-J-Xms4096m
+-J-Xmx4096m
+-J-Xss4M
+-J-XX:ReservedCodeCacheSize=512m
```

Comment:
You don't need to include this changed file since the same change was already merged in another pull request today.